### PR TITLE
Add assignment and abort statements

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

File: modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java [hunk 1]
```diff
@@ -289,7 +294,7 @@ private void addExpressionNode(ExpressionNode expressionNode) {
     public void addLiteralValue(Object value) {
         LiteralNode litExpr = TreeBuilder.createLiteralExpression();
         litExpr.setValue(value);
-        addExpressionNode(litExpr);
+        this.exprNodeStack.push(litExpr);
```
Comment (lafernando) [inline modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java]: Why is this line changed? .. same change below also ..

Comment (anupama-pathirage) [reply modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java]: Reverted to original. This was changed to fix issue with addExpressionNode method which was there previously. It was fixed by #3286 already and hence reverted to the original code. 
Thanks.
